src/box/box.cc

```diff
@@ -1648,6 +1648,22 @@ box_check_wal_queue_max_size(void)
 	return size;
 }
 
```

Comment:
Nit: Please, update the doc bot request. The error msg is "The synchronous transaction queue is full" now.

Reply:
Fixed.